plugins/authentication/ldap/ldap.php

```diff
@@ -177,9 +177,9 @@ public function onUserAuthenticate($credentials, $options, &$response)
 		}
 
 		// Grab some details from LDAP and return them
-		$response->username = $entry->getAttribute($ldap_uid);
-		$response->email    = $entry->getAttribute($ldap_email);
-		$response->fullname = $entry->getAttribute($ldap_fullname) ?: $credentials['username'];
+		$response->username = $entry->getAttribute($ldap_uid)[0] ?? false;
+		$response->email    = $entry->getAttribute($ldap_email)[0] ?? false;
+		$response->fullname = $entry->getAttribute($ldap_fullname)[0] ?? trim($entry->geAttribute($ldap_fullname)) ?: $credentials['username'];
```

Comment:
```suggestion
		$response->fullname = $entry->getAttribute($ldap_fullname)[0] ?? trim($entry->geAttribute($ldap_fullname)[0]) ?: $credentials['username'];
```

Sorry I oversaw that before.